spec/registrant_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+RSpec.describe Registrant do 
+  before(:each) do
+    @registrant_1 = Registrant.new('Bruce', 18, true )
+    @registrant_2 = Registrant.new('Penny', 15 )
+  end
+  
+  describe "Iteration 1" do
+
+    it "exists" do
+
```

Comment:
This space should be removed when you only have one assertion in the test like this and no setup.